Dockerfile_prod

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9
```

Comment:
NIT: By any chance: Did u try the `-slim` variant of the image?

It's only ~45MB instead of ~330MB and I generally prefer those to make things smaller and faster.
Should just work out of the box, but sometimes it doesn't.

```suggestion
FROM python:3.9-slim
```